File: baekjoon/Level/Silver/2847. 게임을 만든 동준이/게임을 만든 동준이.swift
let N = Int(readLine()!)!

var score: [Int] = []
for _ in 0..<N {
    score.append(Int(readLine()!)!)
}

var count = 0
for i in stride(from: N-2, through: 0, by: -1) {
    while score[i] >= score[i+1] {
        score[i] -= 1
        count += 1
    }
}

print(count)

File: baekjoon/Level/Silver/2847. 게임을 만든 동준이/게임을 만든 동준이.py
N = int(input())

score = []
for _ in range(N):
    score.append(int(input()))

count = 0
for i in range(N-2, -1, -1):
    while score[i] >= score[i+1]:
        score[i] -= 1
        count += 1

print(count)